{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2023-37788"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "goproxy v1.1 was discovered to contain an issue which can lead to a Denial of service (DoS) via unspecified vectors."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://github.com/elazarl/goproxy/issues/502",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/elazarl/goproxy/issues/502"
        },
        {
          "name": "https://github.com/elazarl/goproxy",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/elazarl/goproxy"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2023-07-18T19:49Z",
  "publishedDate": "2023-07-18T19:15Z"
}